docs/content/en/docs/clustermgmt/security/manually-renew-certs.md

```diff
@@ -26,6 +26,16 @@ The following table lists the cluster certificate files:
 |                       | apiserver                |
 |                       | front-proxy-client       |
 
+Commands below can be used for quickly checking your certificates expiring date:
+
+```bash
+# The expiry time of api-server certificate on you cp node
+echo | openssl s_client -connect ${CONTROL_PLANE_IP}:6443 2>/dev/null | openssl x509 -noout -dates
```

Comment:
doesn't kubeadm do this already?
https://kubernetes.io/docs/reference/setup-tools/kubeadm/kubeadm-certs/#cmd-certs-check-expiration

Reply:
Right, kubeadm can do this and cover more certs for cp nodes (require ssh). I think these quick commands are still worth to note for etcd nodes, double checking (check from outside instead from inside) and adding health-checks.